.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Tests
+
+on: pull_request
+jobs:
+  code:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup firefox
+        uses: browser-actions/setup-firefox@latest
+        with:
+          firefox-version: 'latest'
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'yarn'
```

Comment:
If this is suppose to be a cache key, it should be more specific to the node and yarn versions. when we update either, we probably want to recreate the cache. Though, I haven't set up any github workflows so I might be missing information,

Reply:
It's a configuration enum. It specifies what caching does it need to use. https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data